### PR TITLE
chore: update beta-5

### PR DESCRIPTION
**Pull request by pedronauck**

### CODE REVIEW

File: deployment/Dockerfile [hunk 1]
```diff
@@ -4,7 +4,7 @@ FROM node:20-slim AS base
 
 # Expose the ENVs to the env of the container
 ENV PORT="${PORT}"
-ENV FUEL_PROVIDER_URL="${FUEL_PROVIDER_URL}"
+ENV FUEL_PROVIDER_BETA5="${FUEL_PROVIDER_BETA5}"
```
Comment (luizstacio) [inline deployment/Dockerfile:7]: Why we change the name of the provider here?

**Review by luizstacio — CHANGES_REQUESTED** [review]

File: packages/app/src/systems/Transaction/component/TxScreen/TxScreenSimple.tsx [hunk 2]
```diff
@@ -132,19 +132,13 @@ export function TxScreenSimple({ transaction: tx, isLoading }: TxScreenProps) {
...
                   <LoadingBox className="w-28 h-5 mt-1" />
                 </>
               }
-              regularEl={
-                <>
-                  Gas used: {formatZeroUnits(tx.gasUsed || '')}
-                  <br />
-                  Gas limit: {formatZeroUnits(tx.gasLimit || '')}
-                </>
-              }
```
Comment (luizstacio) [inline packages/app/src/systems/Transaction/component/TxScreen/TxScreenSimple.tsx:147]: Gas limit is available for transaction of type Script, you can find it on the policies array.

File: deployment/Dockerfile (hunk 1 above)
Comment (LuizAsFight) [reply deployment/Dockerfile:7]: `FUEL_PROVIDER_URL` was better IMO to avoid needing to change later.

**Review by LuizAsFight — CHANGES_REQUESTED** [review]
in transaction screen, can we hide the fee block when it's a mint transaction?

**Pull request merged**